<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QComboBox" name="cbox_paths"/>
        </item>
        <item>
         <widget class="QPushButton" name="btn_refresh">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGroupBox" name="gbox">
        <property name="title">
         <string>基础功能</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QPushButton" name="btn_getSummonerInfo">
             <property name="text">
              <string>获取召唤师信息</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_title_name">
             <property name="text">
              <string>名字:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_name">
             <property name="text">
              <string>未知</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_title_level">
             <property name="text">
              <string>等级:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_level">
             <property name="text">
              <string>0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_title_xp">
             <property name="text">
              <string>经验:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_xp">
             <property name="text">
              <string>0 / 0</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="textEdit_json"/>
         </item>
         <item>
          <widget class="QPlainTextEdit" name="textEdit_result">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="btn_createFromJson">
             <property name="text">
              <string>根据Json创建</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btn_create5V5Tutorial">
             <property name="text">
              <string>创建5V5训练模式</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="1" colspan="2">
              <widget class="QSpinBox" name="sbox_queueId">
               <property name="maximum">
                <number>10000</number>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QComboBox" name="cbox_increment">
               <item>
                <property name="text">
                 <string>不递增</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>+1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>+10</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="0" column="5">
              <widget class="QLabel" name="lbl_currentId">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="text">
                <string>0</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item row="0" column="4">
              <widget class="QLabel" name="lbl_title_currentId">
               <property name="text">
                <string>当前ID</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="lbl_title_queueId">
               <property name="text">
                <string>QueueId</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0" colspan="6">
              <widget class="QPushButton" name="btn_createFromQueueId">
               <property name="text">
                <string>根据ID创建</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="widget_queues" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_setUID"/>
    <addaction name="separator"/>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_setUID">
   <property name="text">
    <string>输入UID</string>
   </property>
  </action>
  <action name="action_about">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
